materials/version-control-R.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: page
+element: notes
+title: Version Control
+language: R
+--- 
+
+> Remind students to setup a GitHub account and email me their username.
+
+> Arrange for partner to `push` following code for 'Collaborating' demo.
+
+```
+get_size_class_ts_data <- function(df){
+  # Convert individual data to time-series data for each of a set of size classes
+  # Input: data frame with a year column for time
+  #        and a size_class column
+  ts_data <-
+    df %>% 
+    group_by(year, size_class) %>% 
+    summarize(counts = n())
+  return(ts_data)
+}
+
+plot_ts_data <- function(df){
+  # Plot time-series data by size class
+  # Input: data frame with year, size_class, and counts columns
+  ggplot(df, aes(x = year, y = counts, color = size_class)) +
+    geom_line()
+}
+```
+
+## Introduction
+
+### Motivation
+
+* Who has a directory on their computer with a bunch of filenames
+    * [Like this?](http://www.phdcomics.com/comics/archive.php?comicid=1531)
+    * [Or like this?](http://www.phdcomics.com/comics/archive.php?comicid=1323)
```

Comment:
Add note in the prep section to open these pages in a browser and zoom in to make the images fill the screen.